src/bandersnatch/delete.py

```diff
@@ -75,7 +94,12 @@ async def delete_packages(config: ConfigParser, args: Namespace, master: Master)
             logger.error(f"{json_full_path} does not exist. Pulling from PyPI")
             await get_latest_json(master, json_full_path, config, executor, False)
             if not json_full_path.exists():
-                logger.error(f"Unable to HTTP get JSON for {json_full_path}")
+                logger.error(
+                    f"Unable to HTTP get JSON for {json_full_path}, "
+                    f"blob files will not be cleaned."
+                )
+                await delete_simple_page(simple_path, canon_name, dry_run=args.dry_run)
+                await delete_simple_page(simple_path, package, dry_run=args.dry_run)
```

Comment:
So, what's the logic here - I'm missing some context:
- If no JSON exists, we should delete the simple pages for a cleanup?

Reply:
I encountered some cases that I didn’t enable json_save and I wanted to clean a package that was deleted in pypi.org , the delete command would raise an error since json file was not present and cannot be get in pypi.org .

This is some kind of quick fix for this case( just delete the simple page, ignore packages completely), my initial thought was that bandersnatch verify would take care of those packages that does not belong to any package. But I later came to another solution: I can “generate” package info from simple page instead of json page.

How do you think we should we handle this case?

Reply:
I feel here we should know (by the config) that we don't save JSON and either pull from the internet or we can just cleanup I guess. I'm not sure I 100% follow this flow.